cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,33 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPageObject from "../support/pages/signUp.pageObject";
+import UserPageObject from "../support/pages/user.pageObject";
+import HomePageObject from '../support/pages/home.pageObject';
+const signUpPage = new SignUpPageObject();
+const homePage = new HomePageObject();
+const userPage = new UserPageObject();
 
 describe('User', () => {
-  before(() => {
-
+  beforeEach(() => {
+    let user;
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = new Object(generateUser);
```

Comment:
The use of `new Object(generateUser)` is unnecessary. You can directly assign `user = generateUser;` since `generateUser` is already an object.